src/PageContainer/component.js

```diff
@@ -100,8 +100,10 @@ export default class PageContainer extends Component {
     }
 
     const item = find(context.collection, pageUrl)
-
+    console.log(isClient)
+    console.log(typeof window !== "undefined")
```

Comment:
We have react-redux connect to pass those via props.

So using `typeof window !== 'undefined'` directly right ?